accepted/global-json-updates.md

```diff
@@ -0,0 +1,790 @@
+      * Otherwise, use the lowest higher major, it's lowest minor, and it's lowest feature band, and the highest patch within that feature band.
+    * `latestPatch`
+      * Even if the requested major, minor, feature band and patch is found, do not use it unless it matches rules below.
+      * Select the highest available patch version that matches the specified major, minor and feature band versions.
+    * `latestFeature`
+      * Even if the requested major, minor, feature band and patch is found, do not use it unless it matches rules below.
+      * Select the highest available feature band that matches the specified major and minor version, and select it's highest patch.
+    * `latestMinor`
+      * Even if the requested major, minor, feature band and patch is found, do not use it unless it matches rules below.
+      * Select the highest available minor that matches the specified major version, and select it's highest feature band, and that feature band's highest patch.
+    * `latestMajor` 
+      * Even if the requested major, minor, feature band and patch is found, do not use it unless it matches rules below.
+      * Select the highest available major version, and select it's highest minor version, and that minor's highest feature band, and that feature band's highest patch.
+    * `disable`
+      * Do not roll forward. Only bind to specified version. This policy means the latest patches, including security patches will not be used.
+    * Defaults: 
+      * If a version is specified in the `global.json`, the default is `patch` to match today's behavior.
+      * If there is no `global.json` or a version is not specified, the default is`latestMajor`, again to match today's behavior.
+ 
+### Examples to illustrate roll-forward options
+
+The following table considers various SDK version combinations that may be available on the machine, and the outcome of the following `global.json`:
+
+```json
+{
+  "sdk": {
+    "version": "2.1.501",
+    "rollForward": [entry in table header]
+  }
+}
+```
+
+| Available SDKs                                       | patch     | feature | minor   | major   | latestPatch | latestFeature | latestMinor | latestMajor | disable |
+|------------------------------------------------------|-----------|---------|---------|---------|-------------|---------------|-------------|-------------|---------|
+| 2.1.500                                              | fail      | fail    | fail    | fail    | fail        | fail          | fail        | fail        | fail    |
+| 2.1.501, 2.1.503                                     | _2.1.501_ | 2.1.501 | 2.1.501 | 2.1.501 | _2.1.503_   | 2.1.503       | 2.1.503     | 2.1.503     | 2.1.501 |
+| 2.1.503, 2.1.505, 2.1.601, 2.2.101, 3.0.100          | 2.1.503   | 2.1.503 | 2.1.503 | 2.1.503 | 2.1.505     | 2.1.601       | 2.2.101     | 3.0.100     | fail    |
```

Comment:
I believe that .503 should be replaced by .505 for this row. .503 would not be selected by any of the policies.